mongodb_migrations/cli.py

```diff
@@ -47,12 +47,12 @@ def run(self):
             print("No previous migrations found")
 
         sys.path.insert(0, self.config.mongo_migrations_path)
-        {
-            Execution.UPGRADE: self._domigrate,
-            Execution.DOWNGRADE: self._dorollback
-        }[self.config.execution]()
+        if self.config.execution is Execution.UPGRADE:
+            self._upgrade()
+        else:
```

Comment:
Open else can be tricky! there is no check whether the person wanted to do a downgrade or not